.github/scripts/randcurl.pl

```diff
@@ -53,20 +55,21 @@ sub getoptions {
     my @all = `$curl --help all`;
     for my $o (@all) {
         chomp $o;
-        if($o =~ /^ -(.), --([^ ]*) (.*)/) {
+        if ($o =~ /^ -(.), --([^ ]*) (.*)/) {
             storedata($1, $2, $3);
         }
-        elsif($o =~ /^     --([^ ]*) (.*)/) {
+        elsif ($o =~ /^     --([^ ]*) (.*)/) {
```

Comment:
Majority of existing code uses `if(`, I wonder if it'd make sense standardizing on it.
It also rhyms with the C-style used in curl.